### ISSUE TITLE
Admin home page

### ISSUE THREAD

**Issue by jakehockey10** [jakehockey10/clothes_casts_and_care#14]
Admins need a central location that provides navigation to the rest of the admin pages.  URL will be (root)/admins/home.

- Feedbacks (admins can only see all or see one, guests access the new, create, and potentially edit, update, and destroy)
    - index
    - show
- Blog posts
    - index
    - Can't see a reason to include show here, but just in case, think about it.
    - new
    - create
    - edit
    - update
    - destroy
- Uploads (these might not be a resource in and of themselves.  This might be the patterns resource...)
    - index
    - show
    - new
    - create
    - destroy

**Comment by jakehockey10** [comment]
I'm contemplating changing the scope of this to something else.  Instead of a homepage, I'm testing out a menu that is always on the right of the window when you are an admin that is signed in.  Right now, the logout button is the only thing there.  But later there could be more.

**Comment by clothescastscare** [comment]
Yeah, I like that menu idea. I also like the set up you have for the admin page :)
